infra/services/envfiles/worker.env.tpl

```diff
@@ -1,4 +1,4 @@
-CONCOURSE_TSA_HOST="${tsa_host}"
+CONCOURSE_TSA_HOST="${tsa_host}:${ssh_port}"
 CONCOURSE_WORK_DIR="/concourse-work-dir"
```

Comment:
In order for this to work, you'll need to pass the var through when templating. This is done here: https://github.com/infrablocks/end-to-end-concourse-ci/blob/1551df48a974b356e8fc3f2b012773e10fc14305/infra/services/secrets.tf#L95 .